src/index.js

```diff
@@ -37,7 +37,12 @@ function buildValidations(referenced, dereferenced, receivedOptions) {
     const schemas = {};
 
     const basePaths = dereferenced.servers && dereferenced.servers.length
-        ? dereferenced.servers.map(({ url }) => new URL(url).pathname)
+        ? dereferenced.servers.map(({ url, variables = {} }) => {
+            Object.keys(variables).forEach((key) => {
+                url = url.replace(new RegExp(`{${key}}`), variables[key].default || '');
+            });
+            return url.slice(0, 1) === '/' ? url : new URL(url).pathname;
```

Comment:
it's more correct to look for  the existence of `//` as can seen [here](https://swagger.io/docs/specification/api-host-and-base-path/)

Reply:
thanks for the changes.

can you please add also these test cases?

Reply:
all cases are covered by existing tests